frontend/src/modules/details/view.jsx

```diff
@@ -451,15 +449,12 @@ const renderDetailImage = (params, data) => {
   return imageNotFound;
 };
 
-UIStore.update((e) => {
-  e.disclaimer = null;
-});
-
 const DetailsView = ({
   match: { params },
   setStakeholderSignupModalVisible,
 }) => {
-  const { profile, countries, loading } = UIStore.currentState;
+  const globalState = UIStore.useState();
```

Comment:
https://lostpebble.github.io/pullstate/docs/use-store-state-hook#use-a-sub-state-of-a-store